hackathon_site/dashboard/frontend/src/components/teamDetail/TeamActionTable/TeamActionTable.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+    Grid,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableRow,
+    Button,
+} from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import React from "react";
+import NotificationImportantIcon from "@material-ui/icons/NotificationImportant";
+import CallSplitIcon from "@material-ui/icons/CallSplit";
+import MergeTypeIcon from "@material-ui/icons/MergeType";
+import DeleteIcon from "@material-ui/icons/Delete";
+
+import styles from "./TeamActionsTable.module.scss";
+
+const TeamActionTable = () => {
+    return (
+        <Grid container spacing={1} direction="column" item md={6} xs={12}>
+            <Grid item>
+                <Typography variant="h2">Actions</Typography>
+            </Grid>
+            <TableContainer component={Paper}>
+                <Table
+                    size="small"
+                    style={{
+                        minWidth: 360,
+                        height: 256.8,
```

Comment:
just curious, what was the intention here for width and height? because I don't think you need it

Reply:
before i had each table wrapped in a grid item, which automatically made the two the same height. after i removed the grid item i just hardcoded in the height of the larger table so that they would still be the same size

the minWidth is so that the table becomes scrollable on small screens. stackoverflow told me this, idk why it works tho lol